src/test/java/teammates/test/cases/browsertests/FeedbackRankQuestionUiTest.java

```diff
@@ -176,35 +176,35 @@ public void testInstructorSubmitAndResultsPage() throws Exception {
 
         InstructorFeedbackResultsPage instructorResultsPage =
                 loginToInstructorFeedbackResultsPageWithViewType("instructor1", "instructor", false, "question");
-        clickAjaxLoadedPanelAndWaitForExpansion(instructorResultsPage, "panelHeading-1", "ajax_auto");
+        instructorResultsPage.clickAjaxLoadedPanelAndWaitForExpansion("panelHeading-1", "ajax_auto");
```

Comment:
If those are two types of panels in the eye of the user, it's fine to have two corresponding methods in the API. Internally both methods can call the same private method with the different id parameters.